### DISCUSSION TITLE
Change Base/Target Branch for PRs on GitHub Mobile

### DISCUSSION THREAD

**Discussion by nvgokhale** [community/community#15003] [Mobile] [answered]
Hello! Just another quick feature request: on the GitHub site, changing the base branch for which a PR merges into is quite simple (just have to click edit and select another branch from the drop down). My request is to bring this feature to mobile and allow a user to change the branch a pull request in trying to be merged into within the app. Thank you!

**Comment by stevepopovich** [comment, marked as answer]
Hi Nidhish! Thanks for the feedback! I filed the request and passed it on to the team for prioritization.
